### PR TITLE
bitbox02: remove Random button / feature

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/components/devices/bitbox02/settings.tsx [hunk 1]
```diff
@@ -17,7 +17,6 @@
 import { Component, h, RenderableProps } from 'preact';
 import { load } from '../../../decorators/load';
 import { translate, TranslateProps } from '../../../decorators/translate';
-import RandomNumber from '../../../routes/device/settings/components/randomnumber';
```
Comment (thisconnect) [inline frontends/web/src/components/devices/bitbox02/settings.tsx:20]: delete the file too `frontends/web/routes/device/settings/components/randomnumber.jsx`

**Review by thisconnect — CHANGES_REQUESTED** [review]

File: frontends/web/src/components/devices/bitbox02/settings.tsx (hunk 1 above)
Comment (benma) [reply frontends/web/src/components/devices/bitbox02/settings.tsx:20]: Can't, it's also used by BB01.

Comment (thisconnect) [reply frontends/web/src/components/devices/bitbox02/settings.tsx:20]: Just read the title now :) . All good then I guess. cc @douglasbakkum

**Review by thisconnect — APPROVED** [review]

File: frontends/web/src/components/devices/bitbox02/settings.tsx (hunk 1 above)
Comment (douglasbakkum) [reply frontends/web/src/components/devices/bitbox02/settings.tsx:20]: 🥀

**Comment by L11R** [comment]
@benma are there any explanations why did you remove this? Did it somehow compromises random generator?

**Pull request merged**